dxw/analytics-with-consent
# Group like settings together

## Description of changes

Reorder settings page to group like settings together

## Screenshots

### Before

![Screenshot 2024-11-28 at 15 16 37](https://github.com/user-attachments/assets/fde33cdf-ae00-480d-826c-e7cc2fdeb7e7)

### After

![Screenshot 2024-11-28 at 15 16 23](https://github.com/user-attachments/assets/c6a60674-93d4-4455-a17c-4c49557e3e97)

### Testing

1. Spin up a local site with this plugin
2. Visit http://localhost/wp-admin/options-general.php?page=analytics-with-consent
3. Note the order of the text fields
2. Checkout this branch
3. Refresh the page
4. Note that analytics settings are now grouped together

## Checklist
